sqlglot/expressions.py

```diff
@@ -1399,6 +1399,10 @@ class ColumnDef(Expression):
     def constraints(self) -> t.List[ColumnConstraint]:
         return self.args.get("constraints") or []
 
+    @property
+    def kind(self) -> t.Optional[DataType]:
+        return self.args.get("kind")
```

Comment:
Let's clean up some places where we do `expr.args.get("kind")`, I believe there are only three (transforms.py, databricks.py, postgres.py)